WELCOME.md

```diff
@@ -0,0 +1,70 @@
+# Welcome to Midjourney
+Midjourney is a new research lab focused on new mediums and tools for empowering people.
+
+ ### Code of Conduct
+- Don't be a jerk
+- Don't use our tools to make images that could inflame, upset, or cause drama. That includes gore and adult content.
+- Be respectful
+
+Make sure you are familiar with the [Full Content Guidelines](unacceptable-content-and-moderation.md).  
+
+You can also find more detail within the Midjourney Discord server in the #rules, #faq, #announcements, and #status channels.
+
+<br/>
+
+---
+--- 
+
+<br/>
+
+
+## Create your First Image 
+Once you have joined the Discord server you will be able to create a limited number of images before needing to subscribe. 
+
+#### 1. Select one of the `#Newbies` channels from the sidebar.
+![Select a Newbies Channel](https://user-images.githubusercontent.com/105028755/167752981-596a4819-163b-4c4c-9241-adfd6231a1f4.jpg)
+
+<br/>
+
+
+#### 2. Use the /Imagine command
+
+To use the Midjourney Bot use a [Discord Slash Command](https://support.discord.com/hc/en-us/articles/1500000368501-Slash-Commands-FAQ) and type`/imagine`.
+Immediately after typing `/imagine` press spacebar, tab, or enter to create the **prompt** field.
+Slash Commands will only work in *Bot Channels*. Typing `/Imagine` will not work in `#Support`, `#Discussion,` and other general channels.
```

Comment:
newbies can't see discussion channel channel so I would remove the reference to it. Also maybe add something that if you have any questions go to #support channel